pkg/pillar/nireconciler/genericitems/dnsmasq.go

```diff
@@ -123,12 +122,9 @@ type DNSServer struct {
 	// ListenIP : IP address (assigned to Dnsmasq.ListenIf) on which the DNS server
 	// should listen.
 	ListenIP net.IP
-	// UplinkIf : uplink interface used to contact UpstreamServers.
-	// Optional argument, leave zero value for NI without uplink.
-	UplinkIf NetworkIf
-	// UpstreamServers : list of IP addresses of public DNS servers to forward
-	// requests to (unless there is a static entry).
-	UpstreamServers []net.IP
+	// UpstreamServers : list of external DNS servers to forward requests to
```

Comment:
The old comment (and type) had this as IP addresses. Can the list now include the names of the DNS servers? Or is it merely a type change without a change in use?